indico/modules/events/editing/operations.py

```diff
@@ -196,25 +196,34 @@ def create_submitter_revision(prev_revision, user, files):
     return new_revision
 
 
-def _ensure_revision_is_undoable(revision):
+def _ensure_revision_can_be_updated(revision):
     if (revision != revision.editable.latest_revision or
             revision.type in (RevisionType.new, RevisionType.ready_for_review)):
         raise InvalidEditableState
 
 
 @no_autoflush
 def undo_review(revision):
-    _ensure_revision_is_undoable(revision)
+    _ensure_revision_can_be_updated(revision)
     revision.editable.published_revision = None
     db.session.flush()
     revision.is_undone = True
     db.session.flush()
     logger.info('Revision %r review undone by %r', revision.editable.editor)
 
 
+@no_autoflush
+def update_review_comment(revision, text):
+    _ensure_revision_can_be_updated(revision)
+    revision.comment = text
+    revision.modified_dt = now_utc()
+    db.session.flush()
+    logger.info('Review comment on revision %r updated by %r', revision, revision.editable.editor)
```

Comment:
Unrelated, but why are we not logging this stuff in the event log as well? (maybe something for a new PR, since it's missing in case of undoing as well AFAICT)

Reply:
I can add it in this one.. should be fairly simple

Reply:
fine for me as well

Reply:
the main reason why i suggested a separate PR is that there may be a bunch of other places where logging is missing, and it's a separate change (w/ a separate changelog entry if you want to add one for it)

Reply:
true, it's actually missing in pretty much every operation (except delete editable). i'll move it to a separate PR to make it neater then